.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,66 @@
+# This needs all the previous artifacts to be created:
+# the image should be published, the repo should be tagged, the helm
+# chart pushed. It's assumed everything is working, and we are doing
+# a last final test at the release creation time to _ensure_ we can
+# release. At this point it's still time to back off in case of problem,
+# not publish the release and iterate on tags.
+
+name: Smoke test
+
+on:
+  push:
+    branches:
+      - '**'
+  release:
+    types: created
+
+jobs:
+  deploy-manifests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+
+      - name: Workaround "Failed to attach 1 to compat systemd cgroup /actions_job/..." on gh actions
+        run: |
+          sudo bash << EOF
+              cp /etc/docker/daemon.json /etc/docker/daemon.json.old
+              echo '{}' > /etc/docker/daemon.json
+              systemctl restart docker || journalctl --no-pager -n 500
+              systemctl status docker
+          EOF
+
+      - name: Create 5 node kind cluster
+        uses: helm/kind-action@master
+        with:
+          config: .github/kind-cluster.yaml
+
+      - name: Deploy kured on default namespace with its helm chart
+        run: |
+          curl -o helm3.tgz https://get.helm.sh/helm-v3.3.0-linux-amd64.tar.gz
```

Comment:
Maybe update to latest?

Reply:
Sure. Will convert this to the the latest installation through script: `curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash`.

Reply:
:+1: